src/commands/report.ts

```
@@ -0,0 +1,397 @@
+    // Generate report output
+    let output: string;
+    try {
+      switch (options.format) {
+        case 'json':
+          output = JSON.stringify(reportData, null, 2);
+          break;
+        case 'html':
+          output = await generateHTMLReport(reportData, options.template);
+          break;
+        case 'markdown':
+        default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)